draft-schedulers.mkd

```diff
@@ -158,6 +178,8 @@ the presentation of the packet schedulers. We consider that a path is active as
 long as it supports the transmission of packets. A Multipath TCP subflow over
 which any of the FIN or RST flags were set is not considered as an active path.
 
+[comment]: # (MA: What about RTO?)
```

Comment:
I think that would be input into the scheduler. A scheduler can consider a path as inactive after 1, 2, ... X RTOs.

Reply:
Agreed, we need to clarify what is an active subflow.